format/mbr/mbr.go

```diff
@@ -0,0 +1,22 @@
+package mbr
+
+import (
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/format/registry"
+	"github.com/wader/fq/pkg/decode"
+)
+
+func init() {
+	registry.MustRegister(decode.Format{
+		Name:        format.MBR,
+		Description: "Master Boot Record",
+		DecodeFn:    mbrDecode,
+	})
+}
+
+func mbrDecode(d *decode.D, in interface{}) interface{} {
+	d.FieldRawLen("code_area", 446*8)
+	d.FieldRawLen("partition_table", 64*8)
```

Comment:
Know if this partition table format is used outside mbr? if not i guess maybe not worth having it as separate format

Reply:
👍